### ISSUE TITLE
Include download links on clients page?

### ISSUE THREAD

**Issue by robinlinden** [Tox/tox.chat#129]
Is this desirable? 
Alternatively, including short descriptions and/or images on download page?
Would help people decide on and download a client without navigating back and forth between the two pages and having to remember the name of the client they like the look of.

**Comment by nurupo** [comment]
That kind of duplicates the purpose of the download page though. The clients page would become the superior version of the downloads page. Maybe we should reconsider the design of Download and Client pages? As proposed in #120, I want the Download page not to list all supported platforms, but to list clients instead. With such idea in the mind, maybe we could somehow merge Download and Client pages?

**Comment by robinlinden** [comment]
You're right. I think we should reconsider the design and merge the pages. Having all the information together with the download would be a big improvement for ease-of-use. 
I read through #120 and I think overall, it's a great idea. What we could do to make the clients-page ready to be merged is add description and screenshots to the Jinja2 template:

```
clients =
[
    {
        'name' : 'qTox',
        'description' : 'A Qt graphical user interface for Tox.'
        'screenshots' : ['qtox_mac.png', 'qtox_linux.png'],
        'platforms' : ['Windows', 'Linux', 'OS X'],
        'Windows' :
        {
            'stable' :
            {
                '32-bit' : 'https://example.org/qtox-stable-32.exe',
...
```
and potentially add a Get Involved page for all the technical stuff users don't care about such as who's maintaining it, what language it's written in, along with GitHub links and things like that.
